main/guides/getting-started/start-a-project.md

```diff
@@ -1,139 +1,177 @@
+`docker run hello-world`
 
-We'll cover [deploying smart contracts](./deploying.md)
-in detail later.
+![031](./assets/031.png)
 
-## Start the Dapp User Interface
 
-The web user interface communicates with the API in the solo client as well as the wallet.
+# Installing the Sample Dapp
+Now you'll use yarn to pull down the sample dapp. The sample dapp will be placed in a subfolder named `demo`.
+`yarn create @agoric/dapp --dapp-base https://github.com/agoric-labs/ --dapp-template dapp-game-places demo`
 
-```sh secondary style3
-# Terminal 3
-cd demo # if not already there
-cd ui && yarn start
-```
+![024](./assets/024.png)
+
+
+# Install Dapp Dependencies
+Now navigate to the `demo` directory and run the `yarn install` command to install any solution dependencies.
+`cd demo`
+`yarn install`
+
+![033](./assets/033.png)
+
+
+# Starting the Network
+Now go ahead and start the network using the `yarn start` command.
+`yarn start:docker`
+
+![034](./assets/034.png)
+
+Once the network has started you can check the logs. Once you see messages showing blocks with a status of `commit` you can rest assured the network is running properly.
+`yarn docker:logs`
+
+![035](./assets/035.png)
+
+
+# Starting the Dapp Smart Contract
+Exit the log window and start the smart contract by running the `yarn start` command.
+`yarn start:contract`
+
+![036](./assets/036.png)
+
+
+# Installing Keplr Wallet
+Next, you'll install the Keplr wallet plug-in. Open up your browser and navigate to [https://www.keplr.app/download](https://www.keplr.app/download). Select the version appropriate to your browser. 
+
+![037](./assets/037.png)
+
+Once the plug-in has been installed, open Keplr and select the option to "Import an existing wallet". Then choose the option to "Use recovery phrase or private key".
+
+![038](./assets/038.png)
 
-Leave this running in its own terminal window and visit [http://localhost:3000](http://localhost:3000) in a web browser.
+![040](./assets/040.png)
 
-## Connect the Dapp to the Agoric Wallet
+To import your wallet, you'll need to copy your mnemonic phrase into Keplr. You can find this series of 24 words back on your terminal window. Copy from this window into your Keplr wallet, then hit the "Import" button. Note that your phrase will NOT be the same as the one shown in this guide, all phrases are unique!
 
-1. Once here, you will be asked to enable the Dapp in your Agoric wallet.
+![039](./assets/039.png)
 
-![Dapp card store ui - needs approval](./assets/must-enable-dapp.png)
+![041](./assets/041.png)
 
-1. Navigate back to [http://127.0.0.1:8000](http://127.0.0.1:8000) and accept the Dapp's request to connect to your wallet.
+Give your new wallet a name and a password.
 
-![agoric wallet - Dapp approval prompt](./assets/accept-dapp-connection.png)
+![042](./assets/042.png)
 
-2. Navigate back to [the Dapp](http://localhost:3000) and it should load the baseball cards to bid on
+Ensure the "Cosmos Hub" chain is selected, then click "Save".
```

Comment:
I'm pretty sure the choice of chains doesn't matter. Why ensure Cosmos Hub is selected?

Reply:
Thought that was in the original instructions. But if it's not needed, I'll take it out.